common/cmake_modules/GncAddTest.cmake

```diff
@@ -147,36 +155,45 @@ function(gnc_gtest_configure)
   message(STATUS "Checking for GMOCK")
   unset(GMOCK_SRC_DIR CACHE)
   if (GMOCK_ROOT)
+    find_path(GMOCK_SRC_DIR src/gmock-all.cc NO_CMAKE_SYSTEM_PATH
+      PATHS ${GMOCK_ROOT})
+  elseif (GTEST_ROOT)
+    find_path(GMOCK_SRC_DIR src/gmock-all.cc NO_CMAKE_SYSTEM_PATH
+      PATHS ${GTEST_ROOT}/googlemock)
```

Comment:
or ${GTEST_ROOT}/../googlemock

Reply:
Ok, I understand. This should handle the case, that only GTEST_ROOT or GMOCK_ROOT are defined, but not both and GnuCash nevertheless finds both, if they are located next to each other like in googletest repo v1.8+.